### DISCUSSION TITLE
ip login works, domain (via proxy) not

### DISCUSSION THREAD

**Discussion by cekvenich2** [ajenti/ajenti#1424] [Q&A] [answered]
Hi. Nice product! I can log in fine when using ip+port.
But I put an https proxy up front with a subdomain.
No joy.

Help? Hint? I did try the 'dev' console, it says nothing other than POST.
Cheers.

**Comment by kiarn** [comment]
Hello @cekvenich2 

You can set the parameter `trusted_domains` in `/etc/ajenti/config.yml` and then restart `Ajenti` to see if it solves your problem.
Short documentation : http://docs.ajenti.org/en/latest/man/config.html#other-global-parameters

Cheers

Arnaud

**Reply by szef1233** [reply]
0b8091e8ed5a5fc54729bdc337985ba47dd5240f

**Comment by vic-cekvenich** [comment, marked as answer]
I was tying in domain w/o the protocol. Once I added https:// upfront works! Thanks!

**Comment by cekvenich2** [comment]
trying. not tying
